src/Microsoft.Framework.Runtime.Hosting/Json/JsonDeserializerResource.cs

```diff
@@ -7,80 +7,77 @@ internal class JsonDeserializerResource
     {
         internal static string Format_IllegalCharacter(int value)
         {
-            return string.Format("Illegal character {0} {0:X4}.", (char)value, value);
+            return $"Illegal character '{(char)value}' [Unicode Hexadecimal {value:X4}].";
```

Comment:
`hexadecimal` (lowercase), plus maybe put it in round parens `()` instead of mean looking square brackets `[]` :smile: